build

```diff
@@ -3,5 +3,13 @@ VERSION=$(git describe --tags HEAD)
 export GO111MODULE=on
 go get github.com/mitchellh/gox
 gox -os="linux darwin windows" -arch="386 amd64" -ldflags="-s -w -X main.Version=${VERSION}"
+# UPX Install START
+# Install UPX 3.96(Latest for linux) working version
+wget https://github.com/upx/upx/releases/download/v3.96/upx-3.96-amd64_linux.tar.xz
```

Comment:
This will make `build` script un-usable on developer machines especially Mac.
It would be good for devs who use mac if this could be addressed.

Reply:
Done